ZelBack/src/services/appsService.js

```diff
@@ -9953,7 +9964,7 @@ async function syncthingApps() {
             const container = containersData[i];
             const containerDataFlags = container.split(':')[1] ? container.split(':')[0] : '';
             if (containerDataFlags.includes('s')) {
-              const containerFolder = i === 0 ? '' : container.split(':')[1];
+              const containerFolder = i === 0 ? '' : `/appdata${container.split(':')[1].replace(containersData[0], '')}`;
```

Comment:
I do not understand this change.

Reply:
search for createAppVolume method.
const execDIR = `sudo mkdir -p ${appsFolder + appId}/appdata`;

this is where the container data is put.

if on appSpecs they specify ex: '/root|s:/root/test'
/root/test is in reality on folder /appdata/test
and all /root is on /apdata.

This was already tested working now as intended on minecraft app running in development nodes.